app/business_logic/BL_parser.cpp

```diff
@@ -117,6 +141,14 @@ int BL_parse_single_string(std::string str)
             log_message("invalid command on string: " + commandString);
             return -1;
         }
+
+        if (command == hyperdrive)
```

Comment:
deze if hoort hier niet

Reply:
deze hoort in de parse single string function onder de if statement for nocommand